src/raftstore/store/worker/apply.rs

```diff
@@ -260,6 +269,10 @@ fn should_flush_to_engine(cmd: &RaftCmdRequest, wb_keys: usize) -> bool {
         return true;
     }
 
+    if has_exclusive_request(cmd.get_requests()) {
```

Comment:
this method just flush old data.  if cmd has requests in the range of `delete_range`,  does it matter?

Reply:
DeleteRange will not be combined with other commands.